internal/pkg/plugininstaller/ci/installer.go

```diff
@@ -55,5 +55,29 @@ func DeleteCIFiles(options plugininstaller.RawOptions) error {
 		CommitMsg:  deleteCommitMsg,
 	}
 	return gitClient.DeleteFiles(commitInfo)
+}
+
+func PushCIContent(options plugininstaller.RawOptions) error {
```

Comment:
This function is as same as `PushCIFiles`.

<img width="1354" alt="image" src="https://user-images.githubusercontent.com/36830265/186059684-b686e8f5-3b0d-4623-a156-6ec9a5a1e9a5.png">